app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,73 @@
+class TweetsController < ApplicationController
+
+	get '/tweets' do 
+		if logged_in?
+		@tweets = Tweet.all
+		erb :'tweets/index'
+		else 
+			redirect '/login'
+		end
+	end
+
+	get '/tweets/new' do 
+		if logged_in?
+			erb :'tweets/new'
+		else
+			redirect '/login'
+		end
+	end
+
+	post '/tweets' do
+		if params[:content].blank?
+			redirect "/tweets/new"
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.